index.js

```diff
@@ -95,7 +95,7 @@ module.exports = function (options = {}) {
 				return path.resolve(dir, pkg.svelte);
 			}
 
-			const resolved = await this.resolve(importee, importer, { skipSelf: true });
+			const resolved = await this.resolve(importee, importer, { skipSelf: true, ...options });
```

Comment:
not sure if legal to pass in from the outside, but to prevent endless loop it would be better to put skipSelf: true after the spread
```suggestion
			const resolved = await this.resolve(importee, importer, { ...options, skipSelf: true });
```

Reply:
looks like Lukas himself suggested putting it in front, so feel free to ignore this https://github.com/lightyen/typescript-paths/issues/3#issue-1206676674